velox/exec/tests/HashJoinBridgeTest.cpp

```diff
@@ -574,4 +574,44 @@ VELOX_INSTANTIATE_TEST_SUITE_P(
     HashJoinBridgeTest,
     testing::ValuesIn(HashJoinBridgeTest::getTestParams()));
 
+TEST(HashJoinBridgeTest, needRightSideJoin) {
```

Comment:
Not sure how valuable this test it. It seems to repeat the logic in production code. Maybe explicit checks might be better.

```
ASSERT_TRUE(needRightSideJoin(kRight));
ASSERT_TRUE(needRightSideJoin(kFull));
ASSERT_FALSE(needRightSideJoin(kInner));
...
```